spec/services/onestop_id_spec.rb

```diff
@@ -31,30 +31,30 @@
     it 'must not be blank' do
       is_a_valid_onestop_id, errors = OnestopId.validate_onestop_id_string('')
       expect(is_a_valid_onestop_id).to be false
-      expect(errors).to include 'must not be blank'
+      expect(errors).to include 'must not be empty'
     end
     it 'must start with "s-" as its 1st component' do
-      is_a_valid_onestop_id, errors = OnestopId.validate_onestop_id_string('69y7pwu-RetSta', expected_entity_type: 'stop')
+      is_a_valid_onestop_id, errors = OnestopId.validate_onestop_id_string('69y7pwu-RetSta')
       expect(is_a_valid_onestop_id).to be false
-      expect(errors).to include 'must start with "s" as its 1st component'
+      expect(errors).to include 'invalid prefix'
     end
 
-    it 'must include 3 components separated by hyphens ("-")' do
+    it 'must include 3 components' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.